pkg/imgpkg/cmd/copy_repo_src_test.go

```diff
@@ -384,13 +385,18 @@ func TestToRepoBundleContainingANestedBundle(t *testing.T) {
 		subject.registry = fakeRegistry.Build()
 
 		destRepo := fakeRegistry.ReferenceOnTestServer("library/bundle-copy")
-		processedBundle, processedImages, err := subject.CopyToRepo(destRepo)
+		processedImages, err := subject.CopyToRepo(destRepo)
 		require.NoError(t, err)
 
 		require.Len(t, processedImages.All(), 4)
+
+		var processedBundle imageset.ProcessedImage
 		processedImageDigest := []string{}
 		for _, processedImage := range processedImages.All() {
 			processedImageDigest = append(processedImageDigest, processedImage.DigestRef)
+			if _, ok := processedImage.Labels[rootBundleLabelKey]; ok {
```

Comment:
ill channel joao here, we should make a copy of rootBundleLabelKey for tests. otherwise tests do not catch breaking change if rootBundleLabelKey changes.

also i see that this file is under `package cmd`. we should move it to `cmd_test`.

Reply:
Hmm, that was kinda intentional actually.

during test assertions of ensuring that the tarball contains the correct label, I do make a copy of that constant. See https://github.com/vmware-tanzu/carvel-imgpkg/pull/189/files/a114b01fb828548b594390c5739c3a98ecd66be5#diff-21cee4994b10fb08208390667e81d81f7e176e9b62caee769d3c42cc4a9cac42R910

That part of the test you have mentioned, digs into an internal constant to find a reference to make a specific type of assertion. which is that the digest ref is correct.

If the label constant were to change accidentally, a test would fail due to the tarball not containing the correct label. 

> also i see that this file is under package cmd. we should move it to cmd_test.
Every test in that package is currently in `cmd` not `cmd_test`. That is a larger change that I think should be done in a separate PR. There is dependencies in those tests that access private members.

Tell Joao I say hi btw.

Reply:
👍 i think separating that into a dedicated PR is prolly good.